networking/spoke-BU0001A0042.json

```diff
@@ -0,0 +1,256 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "hubVnetResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "The hub networking which it is going to be peering"
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "australiaeast",
+                "canadacentral",
+                "centralus",
+                "eastus",
+                "eastus2",
+                "westus2",
+                "francecentral",
+                "germanywestcentral",
+                "northeurope",
+                "southafricanorth",
+                "southcentralus",
+                "uksouth",
+                "westeurope",
+                "japaneast",
+                "southeastasia"
+            ],
+            "metadata": {
+                "description": "The spokes's regional affinity. All resources tied to this spoke will also be homed in this region.  The network team maintains this approved regional list which is a subset of zones with Availability Zone support."
+            }
+        }
+    },
+    "variables": {
+        "orgAppId": "BU0001A0042",
+        "clusterVNetName": "[concat('vnet-hub-spoke-', variables('orgAppId'), '-00')]",
```

Comment:
templetize this:

ask our users to pass the minimum amount of required args. If possible without confusing them not even the full address prefix.

parameter: appInstanceNumber [3,4] --> add TODO consider mutating this into region instead when we go multi-region.

Address Prefix: 10.24<3>
pip: -0<3> 

Address Prefix: 10.24<4>
pip: -0<4>

Reply:
done